Formula/kubekey.rb

```diff
@@ -0,0 +1,37 @@
+class Kubekey < Formula
+  desc "Installer for Kubernetes and / or KubeSphere, and related cloud-native add-ons"
+  homepage "https://kubesphere.io"
+  url "https://github.com/kubesphere/kubekey.git",
+      tag:      "v1.2.1",
+      revision: "94aa580f7ae60792044eddfa3b3e38b8a109d79e"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # don't dirty the git tree
+    rm_rf ".brew_home"
+
+    git_tree_state = Utils.safe_popen_read("git", "status", "--porcelain").strip.empty? ? "clean" : "dirty"
```

Comment:
Since it'll never be dirty, let's just hardcode it.

Reply:
Will do.